{
  "about": "https://jvndb.jvn.jp/ja/contents/2016/JVNDB-2016-001585.html",
  "title": "libdwarf の dwarf_frame2.c の dwarf_read_cie_fde_prefix 関数におけるサービス運用妨害 (DoS) の脆弱性",
  "link": "https://jvndb.jvn.jp/ja/contents/2016/JVNDB-2016-001585.html",
  "description": "libdwarf の dwarf_frame2.c の dwarf_read_cie_fde_prefix 関数には、サービス運用妨害 (out-of-bounds read) 状態にされる脆弱性が存在します。",
  "identifier": "JVNDB-2016-001585",
  "references": [
    {
      "text": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-2091",
      "source": "CVE",
      "id": "CVE-2016-2091"
    },
    {
      "text": "http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2016-2091",
      "source": "NVD",
      "id": "CVE-2016-2091"
    },
    {
      "text": "http://www.openwall.com/lists/oss-security/2016/01/28/8",
      "source": "関連文書",
      "id": "Subject: Re: an out of bound read is found in libdwarf -20151114"
    },
    {
      "text": "http://www.openwall.com/lists/oss-security/2016/01/19/3",
      "source": "関連文書",
      "id": "Subject: Re:Re: Buffer Overflow in lha compression utility"
    },
    {
      "text": "https://jvndb.jvn.jp/ja/cwe/CWE-119.html",
      "id": "CWE-119",
      "title": "バッファエラー(CWE-119)"
    }
  ],
  "cpe": [
    {
      "text": "cpe:/a:libdwarf_project:libdwarf",
      "version": "2.2",
      "vendor": "libdwarf project",
      "product": "libdwarf"
    }
  ],
  "cvss": [
    {
      "version": "2.0",
      "score": "5.0",
      "type": "Base",
      "severity": "Medium",
      "vector": "AV:N/AC:L/Au:N/C:N/I:N/A:P"
    }
  ],
  "date": "2016-03-07T14:44+09:00",
  "issued": "2016-03-07T14:44+09:00",
  "modified": "2016-03-07T14:44+09:00"
}
